Testing meowbuild build farm

milljm opened the pull request:
WIP. Do not merge. A change only to trigger a build on new system.

Refs #000

moosebuild commented:
Job [App tests (Spack)](https://civet.inl.gov/job/595265/) on 593b36c : invalidated by @milljm

added FFTW, ccache

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/595249/) on 593b36c wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/15946/site)

This comment will be updated on new commits.

moosebuild commented:
Job [App tests (Spack)](https://civet.inl.gov/job/595265/) on 593b36c : invalidated by @loganharbour

moosebuild commented:
Job [App tests (Spack)](https://civet.inl.gov/job/595265/) on 593b36c : invalidated by @milljm

use VTK supplied by Spack. Will probably fail

moosebuild commented:
Job [(meowbuild/spack) Code Coverage](https://civet.inl.gov/job/605638/) on 21ca397 wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/15946/coverage)

moosebuild commented:
Job [Minimum gcc version](https://civet.inl.gov/job/625070/) on b0c6605 : invalidated by @milljm

Commit:
Trigger a build to test Create Linux Stack

Manual builds of libmesh are failing on CentOS 8 using Conda.
Trigger a build for which to share to the others.

DO NOT MERGE

Refs #000

Commit:
Trigger a build

Testing restrictions...

Commit:
Testing moose-libmesh package creation

A recent change to civet_recipes should allow Conda packages to build on
CentOS 7.7 images.

Commit:
Trigger build to test end-user targets

moosebuild commented:
Job [End User Experience (Ubuntu)](https://civet.inl.gov/job/640181/) on 2ce016f : invalidated by @milljm

populated Ubuntu container on all meowbuild machines

Commit:
Trigger a build

Test LLVM 10 target

milljm commented:
I have finished the bulk of testing Spack on meowbuild using this PR. Closing.
